cli/admin/README.md

```diff
@@ -52,8 +52,8 @@ You can run the Pravega Admin CLI as follows:
 ```
 ./bin/pravega-admin
 ```
-You will see an output related to the default configuration parameters available at `conf/admin-cli.properties`. The values can be changed at the properties file (`conf/admin-cli.properties`), or use `config set property=...` to change the value.
-(you may want to change this file according to your setting):
+The config values can be changed at the `conf/admin-cli.properties` to permanently change the config, or use `config set property=...` to change temporarily change the config for command line session only.
```

Comment:
"to change temporarily change" -> "to temporarily change"

Reply:
@dellThejas please, pay attention to the text: `to change temporarily change the config`. The same in the first part of the sentence, you are repeating "change" unnecessarily.